core/src/solvers/pipeline_planner.cpp

```diff
@@ -168,6 +171,11 @@ bool PipelinePlanner::plan(const planning_scene::PlanningSceneConstPtr& from,
 	::planning_interface::MotionPlanResponse res;
 	bool success = planner_->generatePlan(from, req, res);
 	result = res.trajectory_;
+	// optionally compute timing to move the eef with constant speed
+	if (req.max_cartesian_speed > 0.0) {
+		trajectory_processing::setMaxCartesianEndEffectorSpeed(*result, req.max_cartesian_speed,
+		                                                       req.cartesian_speed_end_effector_link);
+	}
```

Comment:
As I understood https://github.com/ros-planning/moveit/pull/2674, you implemented a planning adapter. Hence, there is no reason to explicitly call this function here (and below). It's automatically handled by the adapter - if configured.
The `CartesianPath` planner on the other hand could benefit from this explicit call :wink:

Reply:
max cartesian speed can now be set after configuring the adapter https://github.com/ros-planning/moveit_resources/pull/88